### ISSUE TITLE
Inconsistent use of "key" vs "token"

### ISSUE THREAD

**Issue by kristjan** [LockerProject/Locker#825]
At least on Account Settings > API Key, but maybe elsewhere. FWIW, @tantalor prefers "key"

**Comment by othiym23** [comment]
Agreed it should be consistent. Doesn't really feel like a key to me, but am persuadable pending some more feedback from the community.

**Comment by othiym23** [comment]
clickt buton rong

**Comment by tantalor** [comment]
Both "API Token" and "API Key" are used quite frequently. Google, WordPress, Flickr, and 37signals use "API Key". GitHub uses "API Token".

I lean towards "API Key" to avoid confusion with "token" is the context of oAuth (access tokens, etc).

**Comment by tantalor** [comment]
Wikipedia thinks [API key](http://en.wikipedia.org/wiki/API_key) is the standard term.
